doc/changes/latest.inc

```diff
@@ -73,6 +73,8 @@ Bugs
 
 - Fix bug in :func:`mne.viz.plot_alignment` where ECoG and sEEG channels were not plotted and fNIRS channels were always plotted in the head coordinate frame by `Eric Larson`_ (:gh:`8393`)
 
+- When reading BrainVision raw data, the channel units and types were sometimes not inferred correctly by `Richard Höchenberger`_ (:gh:`8434`)
```

Comment:
```suggestion
- When reading BrainVision raw data, the channel units and types were sometimes not inferred correctly, by `Richard Höchenberger`_ (:gh:`8434`)
```

:)

Reply:
Well we haven't **agreed** on that yet! I specifically removed the comma here to make @cbrnr happy 😅😁 Not gonna touch this one without their approval!! 😅😅😅

Reply:
LOL

Reply:
I'm always watching!